lib/files.js

```diff
@@ -253,6 +263,7 @@ function getComponentPackage(name) {
 }
 
 exports.getYaml = control.memoize(getYaml);
+exports.getYamlWithEnv = control.memoize(getYamlWithEnv);
```

Comment:
> This function seems like you just copy and pasted the function above it. Did you just copy and paste the function above it?

No. One `getYamlWithEnv` uses `temp2env`. It seems to make sense to only use that in cases where you know you want environment variables to be included (e.g., config.yaml but not schema.yaml).

Memoizing the function makes sense for the way that the function is used in sites.js, no?

```
files.getFolders(instanceSitesFolder).forEach(function (site) {
  var dir = path.join(instanceSitesFolder, site),
    siteConfig = files.getYamlWithEnv(path.join(dir, 'config')),
    localConfig = files.getYamlWithEnv(path.join(dir, 'local'));
  // ...
});
```